pkg/eventmonitor/consumers/doc.go

```diff
@@ -0,0 +1,11 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+//go:build linux || windows
+
+// Package consumers contains consumer that can be readily used by other packages without
+// having to implement the EventConsumerHandler interface manually:
+// - ProcessConsumer (process.go): a consumer of process exec/exit events that can be subscribed to via callbacks
+package consumers
```

Comment:
iirc the idiomatic go suggests to avoid using plural in package naming

Reply:
In this case the package is intented to be a collection of consumer objects, `consumer` might be confused with the actual consumer implementation.

Reply:
it won't. package is intended to contain a single or **a collection** of objects.

Reply:
Ok, renamed in that case.

Reply:
Had to revert the rename, it seems having a `consumer.go` file and `consumer` package isn't compatible.